tests/Microsoft.Identity.Test.Integration.netfx/SeleniumTests/InteractiveFlowTests.cs

```diff
@@ -227,6 +239,16 @@ private async Task<AuthenticationResult> RunTestForUserAsync(LabResponse labResp
                     .WithTestLogging()
                     .Build();
             }
+            else if (multiCloudSupport)

```

Comment:
Could you please refactor this into a standalone test (maybe not so many steps like this one, just an Interactive + Silent). This will serve as a the basis for documentation and demo for this feature. As it stands, the test code doesn't show the new pattern well enough.

Reply:
Done